kueue-addon/Makefile

```
@@ -0,0 +1,42 @@
+SHELL :=/bin/bash
+
+all: build
+.PHONY: all
+
+LOCALBIN ?= $(shell pwd)/bin
+KUBECTL?=kubectl
+
+IMAGE_REGISTRY ?= quay.io/haoqing
```

```suggestion
IMAGE_REGISTRY ?= quay.io/open-cluster-management
```

Even if it's only available locally, it might be better to have ocm repo as the registry?

---

yes, should be ocm repo.